marketplace_tracker.js

```diff
@@ -192,8 +268,10 @@ function scanMarketList() {
         }
         // Don't scan the sell container, this is done by scanSellList()
         if (items[0].firstChild.className == "marketplace-sell-container") {
+            highlightFavoriteSell();
```

Comment:
Add this line to `scanSellList()` instead

Reply:
I think it's more efficient to move `scanSellList()` to this if instead.